### DISCUSSION TITLE
Version-Mike

### DISCUSSION THREAD

**Discussion by KarthikModelon** [squidfunk/mkdocs-material#4178] [Q&A]
In materials 8.2.13 the version dropdown is not available but it is working good in 7.0.0. Please update the version dropdown for 8.2.13

**Comment by squidfunk** [comment]
Thanks for reporting. Please provide a reproducible example and attach it as a zip file, or we're not able to help you.

**Reply by KarthikModelon** [reply]
It seems to be working now. Any updates made to fix it?

**Reply by squidfunk** [reply]
Nope, nothing changed.
